src/MapfileStyleParser.ts

```diff
@@ -48,22 +55,232 @@ export class MapfileStyleParser implements StyleParser {
   }
 
   /**
-   * Get the GeoStyler-Style Filter from an Mapfile Expression.
+   * Get the GeoStyler-Style Filter from an Mapfile EXPRESSION and CLASSITEM.
+   *
+   * @param {object} mapfileClass The Mapfile Class
+   * @return {Filter} The GeoStyler-Style Filter
+   */
+  getFilterFromClassItem(mapfileClass: any): Filter | undefined {
+    const classItem: string = mapfileClass.classitem;
+    assert(mapfileClass.classitem, 'Mapfile CLASSITEM undefined!');
+    const expression: string = mapfileClass.expression;
+    switch (expression.charAt(0)) {
+    case '"':
+      return ['==' as ComparisonOperator, mapfileClass.classitemclassItem, expression];
+    case '/':
+      return ['FN_strMatches', classItem, expression];
+      break;
+    case '{':
+      return [
+        'FN_strMatches',
+        classItem,
+        `/${expression.substring(1, expression.length - 1).replace(',', '|')}/`,
+      ];
+    default:
+      break;
```

Comment:
You can remove the default as it does nothing.

Reply:
Check also other switch on this file please

Reply:
Ok, I removed the empty default cases and set the linting to off.